source/lac/vector_memory.inst.in

```diff
@@ -20,3 +20,9 @@ for (VECTOR : VECTOR_TYPES)
     template class VectorMemory<VECTOR>;
     template class GrowingVectorMemory<VECTOR>;
 }
+
+for (SCALAR : REAL_SCALARS)
+{
+    template class VectorMemory<LinearAlgebra::CUDAWrappers::Vector<SCALAR>>;
```

Comment:
For other types we only instantiate things if we have the external dependency. Would it make sense to guard this with ``DEAL_II_WITH_CUDA``?

Reply:
yeah good catch